C_CPP/Data Structures/Standard Template Library/STL-LISTS.md

```diff
@@ -0,0 +1,307 @@
+# STL LIST
+<p> List is a sequence container that allows insertion and deletion operations in constant time from any place in the sequence and can iterate in both directions forward and backward but traversing is bit slower. </p>
+
+<p> Declaration of list in stl</p>
+
+```
+list < int > li = {100,20,30,30,50};
+```
+## Iterator of list :
+<p> Iterator is like the pointer which is to traverse the list and also used to insert or delete any element from any position.</p>
+
+```
+list <int> :: iterator it;
+```
+<p> Code for list important functions </p>
+
+```
+#include <iostream>
```

Comment:
```suggestion

```c++
#include <iostream>
```


Do this slight change in all cpp codes for cpp syntax effect then you'r good to go for merge !!

Reply:
Sir i have done the changes that you proposed now you can merge @vasu-1 .